app/_hub/kong-inc/degraphql/_index.md

```diff
@@ -27,113 +26,127 @@ params:
   name: degraphql
   service_id: true
   dbless_compatible: 'yes'
-  config: null
+  config:
+    - name: graphql_server_path
+      required: false
```

Comment:
I set this as "false" even though the code has it as required. It's been pointed out that entries with a default value that are labelled as required are confusing to users - they're required to set the value, but the value is set? This makes no sense.

Reply:
I think we better stay consistent. A value being required means you cannot set it explicitly to a JSON `null` value. 

I agree it’s not intuitive, but it certainly makes sense.

Reply:
+1 to preserve `required: true`. But yeah... totally agree with you Lena. we should probably how to document this in a more intuitive way :thinking:

Reply:
> we should probably check how to document this in a more intuitive way 🤔

We'll give this bit some more thought. For now, I'll set it to `required: true`.